skeleton/scaffold-navigation/src/main.ext

```diff
@@ -7,7 +7,7 @@ import 'bootstrap';
 // @if feat.typescript
 import {Aurelia} from 'aurelia-framework';
 // @endif
-import environment from './environment';
+import * as environment from '../config/environment.json';
```

Comment:
Both webpack and TypeScript support normal `import environment from './environment.json';`.
And explicit `.json` is only required by TypeScript.

I am not sure why you need `* as`.
Also the original line should be retained in a condition `// @if !feat.webpack` for cli-bundler app to work.

Reply:
`import * as environment ` is needed unless [`allowSyntheticDefaultImports` is set to `true`](https://www.typescriptlang.org/docs/handbook/compiler-options.html). I think I will stick to `* as` syntax instead.

Reply:
Retained the `import environment from './environment';` for cli-bundler